main/guides/zoe/contract-requirements.md

```diff
@@ -103,6 +110,65 @@ Use `zcf.makeInvitation()` to create the first party's `invitation`:
     );
 ```
 
+The third argument (which is optional and wasn't needed for the first `invitation`) says
+the counterparty has to offer an `amount` matching the first party's `want.Price`,
+and shoateArgs) => {
+  ...
+  // your code here
+  return harden({ creatorFacet, creatorInvitation, publicFacet });
+}
+harden(start);
+export { start };
+```
```

Comment:
Because you omitted the opening "```", markdown took this one, which you intended to close the code block, as the opening of a code block and then quoted the following prose thinking it is code, as seen at https://1750ad84.documentation-7tp.pages.dev/guides/zoe/contract-requirements.html

Reply:
Resolving all conversations that were only about the fubar contract-requirements.md